### PR TITLE
The home endpoint and unit test is implemented

### PR DESCRIPTION
**Pull request by DMGochoa**
1. On the api.py is added the endpoint.
2.  The new bank_api/resources/home_resource.py is created
3. The new tests/test_home.py is created

### CODE REVIEW

File: tests/test_home.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            "email": "ca@texample.com",
+            "phone_number": "999555999"
+    } 
+    # User data to login
+    user_creation_response = client.post(f"{USERS_ENDPOINT}",json=user_json)
+    print(user_creation_response.json)
+    login_json = {
+        "username": user_creation_response.json["username"],
+        "password": user_creation_response.json["password"],
+        "code": user_creation_response.json["code"] # Esta pasando algo con el tipo de dato de code, me lo lee cuando es un string y no un int
```
Comment (Camiloarceg) [inline tests/test_home.py]: Hi Diego try commenting on English, please

File: tests/test_home.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    print(login_json)
+    login_response = client.post(f"{LOGIN_ENDPOINT}",
+                                 json=login_json)
+    
+    print(login_response.text)
+    home_response = client.get(f'{HOME_ENDPOINT}',  
+                               headers={'Authorization':login_response.json['access_token']})
+    print(home_response.status_code)
```
Comment (Camiloarceg) [inline tests/test_home.py]: I think is better to not push prints to the main repository, just an observation.

File: tests/test_home.py (hunk 1 above)
Comment (DMGochoa) [reply tests/test_home.py]: I already changed to english, ty camilo

File: tests/test_home.py (hunk 2 above)
Comment (DMGochoa) [reply tests/test_home.py]: I have already removed the prints, ty Camilo

**Review by Camiloarceg — COMMENTED** [review]
No further comments on my end, the code seems good to me.

**Review by kevingaray — APPROVED** [review]
hope this is working

**Review by Camiloarceg — APPROVED** [review]
Approving changes for merging

**Pull request merged**